src/msix/msix.cpp

```diff
@@ -70,6 +70,25 @@ MSIX_API HRESULT STDMETHODCALLTYPE UnpackPackage(
     return static_cast<HRESULT>(MSIX::Error::OK);
 } CATCH_RETURN();
 
+MSIX_API HRESULT STDMETHODCALLTYPE UnpackPackageFromPackageReader(

```

Comment:
>UnpackPackageFromPackageReader [](start = 35, length = 30)

I would prefer to see the other unpacks call into this one, as it is the natural progression.  But not required in this change.

Reply:
Good idea. I'll go ahead and make this change since it should simplify test validation (won't need to write new tests for the new APIs. The existing tests will cover the new API since they'll eventually call into the new APIs)